astroplan/core.py

```diff
@@ -118,17 +158,19 @@ def __init__(self, name=None, location=None, latitude=None, longitude=None,
 
     def altaz(self, time, target=None, obswl=None):
         """
-        If `target` is None, generates an altitude/azimuth frame. Otherwise,
-        calculates the transformation to that frame for the requested `target`.
+        Get an `~astropy.coordinates.AltAz` frame or coordinate.
+
+        If ``target`` is None, generates an altitude/azimuth frame. Otherwise,
+        calculates the transformation to that frame for the requested ``target``.
 
         Parameters
         ----------
-        time : `~astropy.time.Time`
+        time : Anything accepted by `~astropy.time.Time`
```

Comment:
I think it might be better to be a bit more specific about what this means.  The average user might not understand what "accepted by" means.  Maybe just "as the first argument to" is clear enough?  I also think it would be useful to specify that it could be just a `Time` object.  It may not be obvious to everyone that a `Time` accepts a `Time` as a first argument...

(same for all the other places, hopefully that's just a find-and-replace-all, though)